packages/ui-components/src/FormComponents/MetadataEditorTags.tsx

```diff
@@ -197,63 +198,79 @@ export class MetadataEditorTags extends React.Component<
         </button>
       );
     return (
```

Comment:
We should address the codeQL warnings above, I think they can be addressed by some refactoring to use `useEffect` or similar